test/integration/exasol/test_get_metadata_functions.py

```diff
@@ -25,6 +25,10 @@
 ENGINE_SCHEMA_2_DATABASE = "ENGINE_SCHEMA_2_DATABASE"
 
 
+@pytest.mark.parametrize(
+    "engine_name",
+    [ENGINE_NONE_DATABASE, ENGINE_SCHEMA_DATABASE, ENGINE_SCHEMA_2_DATABASE],
+)
```

Comment:
Wow - that option is new to me!
Parametrizing the class and by that making the parameter available to each of the test cases in the class!
:exploding_head: :heart: